files/ko/games/tutorials/2d_breakout_game_pure_javascript/mouse_controls/index.html

```diff
@@ -37,7 +37,7 @@ <h2 id="마우스_움직임에_따라_패들_이동_하기">마우스 움직임
     }
 }</pre>
 
-<p>이 함수에서 우리는 먼저 상대적인 <code>relativeX</code> 를 구합니다. 이 값은 뷰포트(<code>e.clientX</code>)와 캔버스 왼쪽 가장자리 (<code>canvas.offsetLeft</code>) 사이의 거리에 마우스 위치를 뺀 값입니다. 상대적인 X 포인터 위치가 0보다 크고 캔버스 폭보다 적으면 포인터가 경계 내에 있게 되고, <code>paddleX</code>의 위치(패들 왼쪽 가장자리에 배치되어 있다)는 패들 width의 반을 뺀 <code>relativeX</code>의 값으로 설정되므로 이동은 실제로 패들 가운데 상대적으로 이동이 됩니다.</p>
+<p>이 함수에서 우리는 첫번째로 <code>relativeX</code> 값을 계산하는데, 이는 뷰포트 (<code>e.clientX</code>) 안에 있는 가로 마우스 위치 빼기 캔버스의 좌측 끝과 뷰포트 (<code>canvas.offsetLeft</code>) 의 좌측 끝 사이의 거리와 같습니다. 효과적으로 이것은 캔버스의 좌측 끝과 마우스 포인터 사이의 거리와 같습니다. 상대적인 X 포인터 위치가 0보다 크고 캔버스 폭보다 적으면 포인터가 경계 내에 있게 되고, <code>paddleX</code>의 위치(패들 왼쪽 가장자리에 배치되어 있습니다)는 패들 폭의 반을 뺀 <code>relativeX</code>의 값으로 설정되므로 이동은 실제로 패들의 중앙에 상대적일 것입니다.</p>
```

Comment:
```suggestion
<p>이 함수에서 우리는 첫번째로 <code>relativeX</code> 값을 계산하는데, 이는 뷰포트 (<code>e.clientX</code>) 안에 있는 가로 마우스 위치에 캔버스의 좌측 끝과 뷰포트 (<code>canvas.offsetLeft</code>) 의 좌측 끝 사이의 거리를 뺀 값과 같습니다. 이는 캔버스의 좌측 끝과 마우스 포인터 사이의 거리와 같습니다. 상대적인 X 포인터 위치가 0보다 크고 캔버스 폭보다 적으면 포인터가 경계 내에 있게 되고, <code>paddleX</code>의 위치(패들 왼쪽 가장자리에 배치되어 있습니다)는 패들 폭의 반을 뺀 <code>relativeX</code>의 값으로 설정되므로 이동은 패들의 중앙과 관련될 것입니다.</p>
```

몇몇 문장들이 이해하기 어렵게 해석되었습니다. 직역보단 좀 더 이해하기 쉽게 수정해보았습니다.